src/e2e/java/teammates/e2e/cases/lnp/BaseLNPTestCase.java

```diff
@@ -79,6 +85,16 @@
      */
     protected abstract Map<String, String> getRequestParameters();
 
+    /**
+     * Returns the limit of the ratio of the error samples against total number of samples.
+     */
+    protected abstract double getErrorRateLimit();
+
+    /**
+     * Returns the limit of the specified response time in milliseconds.
```

Comment:
Can we change this to seconds?

`Returns the maximum allowable threshold for the mean response time (in seconds) for the test endpoint.`